cmd/evalviewer/README.md

```diff
@@ -0,0 +1,69 @@
+# Eval Viewer
+
+A CLI tool to run evaluations and display results in a nice table format.
+
+## Usage
+
+### Run Command (Recommended)
+Run tests with `GOEVALS=1` and automatically display results:
+
+```bash
+# Run evaluations for conversations package
+go run main.go run -v ./conversations
```

Comment:
nit: maybe put `GOEVALS=1` into the example so that copy/paste works out of the box?

Reply:
Updated all the README examples.